client/src/components/portfolio/CurrentPortfolio.js

```diff
@@ -27,7 +28,9 @@ class CurrentPortfolio extends Component {
         <tr key={transaction.id}>
           <td>{transaction.stockTicker}</td>
           <td>{transaction.units}</td>
+          <td>{transaction.price}</td>
           <td>{transaction.currentPrice}</td>
+          <td>{transaction.status}</td>
```

Comment:
let us rename this to "Profit/Loss" instead of status.

Reply:
Done.